### ISSUE TITLE
[Global Styles] Main gray button's colors are not accessible

### ISSUE THREAD

**Issue by davefuentes** [zurb/foundation-sites#7104]
**How can we reproduce this bug?**

1. Go to Global Styles Page
2. Use WebAIM Color Contrast Checker (http://webaim.org/resources/contrastchecker)
3. Enter "FFFFFF" as "foreground" color.
4. Enter "777777" as "background" color.

**What did you expect to happen?**
Contrast ratio for normal text should be 4.5:1 or higher
WCAG AA criteria should pass

**What happened instead?**
Contrast ratio is below standard.
See: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=777777

**Comment by IamManchanda** [comment]
seems like #515151 will be best color ?

**Comment by davefuentes** [comment]
Hey Harman, 
As long as you cover AA, you are ok.

**Comment by davefuentes** [comment]
Yes, that one definitely should do it:
http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=515151

**Comment by davefuentes** [comment]
Do you want me to create a pull request with this change?

**Comment by zurbrandon** [comment]
Please do! @davefuentes

**Comment by andycochran** [comment]
Even a very slight change to `#767676` lets AA pass.

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
